test/fixtures/dynamic-argument/expected.js

```diff
@@ -1,7 +1,7 @@
 const MODULE = 'test-module';
 
 new Promise(resolve => {
-  resolve(require(MODULE));
+  resolve(require(`${ MODULE }`));
```

Comment:
no need for the spaces inside the curly braces; and in fact this contradicts airbnb's style guide